### DISCUSSION TITLE
Mobile docs dark mode and invalid link in blog post

### DISCUSSION THREAD

**Discussion by ivanjuric** [rails/rails#51398] [Docs]
### Dark mode issue

Edge guides on Android (Firefox and Chrome) seems to have inverted dark mode theme around _More Ruby On Rails_ button.
When mobile phone is in dark mode, background color is white, and when it's not in dark mode, color is dark.
Please see screenshot below.

![Screenshot_20240323_181506_Firefox](https://github.com/rails/rails/assets/7100678/251f0924-b7c0-49fe-a803-028f95a874db)

![Screenshot_20240323_181526_Firefox](https://github.com/rails/rails/assets/7100678/07877254-22c0-4d10-92f0-3552f74ef281)

### Invalid link
Also, just to mention invalid link in blog post: https://rubyonrails.org/2024/3/20/rails-guides-get-a-facelift.
Clicking on **open a discussion on GitHub** leads to following url: https://rubyonrails.org/2024/3/20/[https://github.com/rails/rails/issues](https://github.com/rails/rails/discussions/categories/docs), and results with, of course 404.

<img width="776" alt="image" src="https://github.com/rails/rails/assets/7100678/ab86dcea-3003-409d-9713-d14c0a1345d1">

**Comment by AmandaPerino** [comment]
Hi @ivanjuric thanks for catching the broken link - fixed now.
We are aware of the navbar on mobile. This will be dealt with in a future update. Thank you!

**Comment by AmandaPerino** [comment]
Hi @ivanjuric thanks again for the feedback. The inverted navbar has been fixed in #51444.
